pkg/cloudprovider/aws/suite_test.go

```diff
@@ -27,12 +27,15 @@ import (
 	"github.com/awslabs/karpenter/pkg/packing"
 	"github.com/awslabs/karpenter/pkg/test"
 	. "github.com/awslabs/karpenter/pkg/test/expectations"
+	"github.com/awslabs/karpenter/pkg/utils/filesys"
 	"github.com/awslabs/karpenter/pkg/utils/parallel"
 	"github.com/awslabs/karpenter/pkg/utils/resources"
 	"github.com/patrickmn/go-cache"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 
+	"io/fs"
```

Comment:
nit: this should be below the non-std lib imports

Not sure why there are spaces separating these non-std lib imports, they can be pushed together.